Formula/plantuml.rb

```diff
@@ -15,7 +15,10 @@ def install
     libexec.install "plantuml.#{version}.jar" => jar
     (bin/"plantuml").write <<~EOS
       #!/bin/bash
-      GRAPHVIZ_DOT="#{Formula["graphviz"].opt_bin}/dot" exec java -jar #{libexec}/#{jar} "$@"
+      if [[ "$@" != *"-gui"* ]]; then
```

Comment:
I suspect that this line may not work if you pass more than one argument to `plantuml` (untested, I may be wrong). Could you please test passing more than one argument?

Reply:
Is `-gui` a documented, supported argument for `plantuml`?

Reply:
@MikeMcQuaid 
> Is `-gui` a documented, supported argument for `plantuml`?

Yes